build/csslint/xfail.list

```diff
@@ -53,6 +53,7 @@ apps/settings/style/downloads.css 0 3
 apps/sms/style/root.css 0 1
 apps/sms/style/sms.css 0 4
 apps/system/mobile_id/style/mobile_id.css 0 16
+apps/system/style/devtools/devtools_auth.css 1 0
```

Comment:
@jryans What was the error? On my Mac, it is a parsing error (so not because of csslint) because it's running on xulrunner 34 which doesn't support object-fit. Seems like https://bugzilla.mozilla.org/show_bug.cgi?id=1101331 is needed to update xulrunner.